arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/transients/TStoreSection.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright ConsenSys Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.zktracer.module.hub.section.transients;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static net.consensys.linea.zktracer.module.hub.fragment.TransientFragment.doing;
+import static net.consensys.linea.zktracer.module.hub.fragment.TransientFragment.undoing;
+
+import net.consensys.linea.zktracer.module.hub.Hub;
+import net.consensys.linea.zktracer.module.hub.defer.PostRollbackDefer;
+import net.consensys.linea.zktracer.module.hub.fragment.ContextFragment;
+import net.consensys.linea.zktracer.module.hub.fragment.TransientFragment;
+import net.consensys.linea.zktracer.module.hub.section.TraceSection;
+import net.consensys.linea.zktracer.module.hub.signals.Exceptions;
+import net.consensys.linea.zktracer.runtime.callstack.CallFrame;
+import org.apache.tuweni.bytes.Bytes32;
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+
+public class TStoreSection extends TraceSection implements PostRollbackDefer {
+
+  public TStoreSection(Hub hub) {
+    super(hub, (short) 4);
+
+    final short exceptions = hub.pch().exceptions();
+    final ContextFragment readCurrentContext = ContextFragment.readCurrentContextData(hub);
+
+    this.addStackAndFragments(hub, readCurrentContext);
+
+    if (Exceptions.any(exceptions)) {
+      checkArgument(
+          Exceptions.stackException(exceptions) || Exceptions.outOfGasException(exceptions),
+          "The only possible exceptions are STATICX or OOGX");
+      return;
+    }
+
+    // We are now unexceptional
+    hub.defers().scheduleForPostRollback(this, hub.currentFrame());
+
+    final Address address = ((ContextFragment) fragments().get(1)).getAccountAddress();
+    final Bytes32 storageKey = Bytes32.leftPad(hub.messageFrame().getStackItem(0));
+    final Bytes32 valueCurr =
+        Bytes32.leftPad(hub.messageFrame().getTransientStorageValue(address, storageKey));
+    final Bytes32 valueNext = Bytes32.leftPad(hub.messageFrame().getStackItem(1));
+
+    this.addFragment(doing(hubStamp(), address, storageKey, valueCurr, valueNext));
+  }
+
+  @Override
+  public void resolveUponRollback(Hub hub, MessageFrame messageFrame, CallFrame callFrame) {
+    this.addFragment(undoing(hubStamp(), revertStamp(), (TransientFragment) fragments().get(2)));
```

Comment:
Why not have include e.g. a
```
final TransientFragment tstoreDoing;
```
in the class ? This is brittle (I'm talking about the explicit `2`) compared to referencing `tstoreDoing` directly.